qtfred/src/ui/FredView.cpp

```diff
@@ -582,6 +620,10 @@ void FredView::on_actionBriefing_triggered(bool) {
 	auto eventEditor = new dialogs::BriefingEditorDialog(this);
 	eventEditor->show();
 }
+void FredView::on_actionWaypoint_Paths_triggered(bool) {
+	auto editorDialog = new dialogs::WaypointEditorDialog(this, _viewport);
```

Comment:
Ah wait, the WaypointEditorDialog and related Model files weren't added to the source_groups for the qtfred project.